src/pages/RocketSlice.test.js

```diff
@@ -0,0 +1,55 @@
+describe('reserveRocketAction', () => {
+  const reserveRocketAction = (state, action) => {
+    const id = action.payload.id.toString();
+    const updatedRockets = state.rockets.map((rocket) => {
+      if (rocket.id === id) {
+        return {
+          ...rocket,
+          reserved: !rocket.reserved,
+        };
+      }
```

Comment:
- [ ] Good job Team 😃, however you still need to fix one part 😎 , currently all test are not passing. Kindly fix it 👍 

![Capture](https://user-images.githubusercontent.com/57408419/233792031-e27b9ddc-adab-419c-ad3e-3867d0b4b329.PNG)